docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.3.0-alpine
+FROM node:16-alpine3.17
```

Comment:
had to do this due to a node version requirement in the new @graphprotocol/graph-cli version